### PR TITLE
feat: send script also in exportconfig variable

### PR DESCRIPTION
**Pull request by odinuv**
second step of https://github.com/keboola/data-loader/issues/26

### CODE REVIEW

**Comment by odinuv** [comment]
tests are here https://jenkins.keboola.com/view/Provisioning/job/PHPUnit%20-%20provisioning%20(branch%202)/21/console

**Comment by pivnicek** [comment]
Ok, now I'm a little lost as to why we're doing this :-D.  It looks like duplicate functionality?  Sorry:)

**Comment by pivnicek** [comment]
Aha!  Now I see #111 and it makes sense 👍

File: Entity/Server/Docker.php [hunk 1]
```diff
@@ -800,9 +800,14 @@ public function getCluster()
         return $this->cluster;
     }
 
-    public function setExportConfig(array $inputConfiguration)
+    public function setExportConfig(array $inputConfiguration, $script)
```
Comment (pivnicek) [inline Entity/Server/Docker.php]: do we want to use type hinting?

**Comment by odinuv** [comment]
yes, historically the script was sent through the 'SCRIPT' env variable, which was processed in jupyter init. then we added the option to read the script from configuration, which is done in data-loader. So now i want to get rid of the logic in jupyter which is this https://github.com/keboola/docker-jupyter/pull/4/files because it is untested and nowdays completely duplicated in data-loader (as it now produces a full ipynb file)

File: Entity/Server/Docker.php (hunk 1 above)
Comment (odinuv) [reply Entity/Server/Docker.php]: can't do - php 5.6

Comment (pivnicek) [reply Entity/Server/Docker.php]: aha!  ok

**Pull request merged**